### PR TITLE
Removing outdated screen shots

### PR DESCRIPTION
**Pull request by suearkinunity**
### Purpose of this PR

Removed screenshots that were out of date for ProBuilder 6.

### Links

Several Jira tickets.

### Comments to Reviewers

Documentation only; no text changes either.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=524) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.probuilder?pullRequest=524) before we can accept your contribution.<br/><hr/>**Sue Arkin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.probuilder?pullRequest=524) it.</sub>

**Comment by JonMUnity** [comment]
Should we remove the image files from the repo, too?

**Comment by suearkinunity** [comment]
Yes, sorry. Removed.

**Comment by suearkinunity** [comment]
@modrimkus please can you merge this?

File: Documentation~/menu-editors.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 This sub-menu provides access to tools and editor windows.
 
-![Tools > ProBuilder > Editors menu](images/menu-editors.png)
-
 ## New Bezier Shape
```
Comment (suearkinunity) [inline Documentation~/menu-editors.md:5]:
```suggestion
## Create Bezier Shape
```

File: Documentation~/menu-editors.md [hunk 2]
```diff
@@ -2,17 +2,15 @@
 
 This sub-menu provides access to tools and editor windows.
 
-![Tools > ProBuilder > Editors menu](images/menu-editors.png)
-
-## New Bezier Shape
+## Create Bezier Shape
```
Comment (suearkinunity) [inline Documentation~/menu-editors.md:5]: @JonMUnity there's a comment from May on ticket 5820 that says to change "new" to "create" for three editors, but in my version of U6 + PB6 it still says "new".

Comment (JonMUnity) [reply Documentation~/menu-editors.md:5]: Let's go with what the UI says in that case. Sorry.

Comment (suearkinunity) [reply Documentation~/menu-editors.md:5]:
```suggestion
## New Bezier Shape
```

File: Documentation~/menu-editors.md [hunk 3]
```diff
@@ -2,17 +2,15 @@
...
 
 Use the **Bezier Shape** tool to define a Bezier curve around which ProBuilder extrudes a Mesh.
 
 > **Warning:** This tool is experimental, meaning that it is still under development, and might reduce ProBuilder's stability. Use with caution.
 
 For more information, see the [Bezier Shape tool](bezier.md) documentation.
 
-## New Poly Shape
+## Create Poly Shape
```
Comment (suearkinunity) [inline Documentation~/menu-editors.md:13]:
```suggestion
## New Poly Shape
```

File: Documentation~/menu-editors.md [hunk 4]
```diff
@@ -30,7 +28,7 @@ Use the **Material Editor** window to apply materials to objects or faces.
 
 For more information, see the [Material Editor](material-tools.md) window documentation.
 
-## New Shape
+## Create Shape
```
Comment (suearkinunity) [inline Documentation~/menu-editors.md:31]:
```suggestion
## New Shape
```

**Review by suearkinunity — COMMENTED** [review]
Create > New

**Comment by suearkinunity** [comment]
@JonMUnity actually, since I opened 528 https://github.com/Unity-Technologies/com.unity.probuilder/pull/528/files
This one seems pointless.

**Comment by suearkinunity** [comment]
See PR 528 instead - deals with the same area, but better

**Pull request closed without merging**